app/controllers/application_controller.rb

```diff
@@ -4,8 +4,18 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_app_info_headers
 
+  private
+
   def set_app_info_headers
     headers["X-GitHub-Repository"] = "https://github.com/department-of-veterans-affairs/vets-api"
```

Comment:
it was a thought for determining the project before we talked about single repo, don't think this is a problem to include